PWGHF/TableProducer/treeCreatorLcToK0sP.cxx

```diff
@@ -135,7 +137,8 @@ DECLARE_SOA_TABLE(HfCandCascLites, "AOD", "HFCANDCASCLITE",
                   full::OriginMcRec,
```

Comment:
This flag contains exactly the information about prompt/non-prompt. Why do you need to add another one?